head.go

```diff
@@ -1020,6 +1020,13 @@ func (h *Head) MaxTime() int64 {
 	return atomic.LoadInt64(&h.maxTime)
 }
 
+// compactable returns whether the head has a compactable range.
+func (h *Head) compactable() bool {
+	// The head has a compactable range if 1.5 level 0 ranges are between the oldest
+	// and newest timestamp. The 0.5 acts as a buffer of the appendable window.
```

Comment:
you can combine the func comment with this one to make it only a single comment describing the func

Reply:
ok